public/app/plugins/datasource/opentsdb/datasource.ts

```diff
@@ -306,6 +306,10 @@ export default class OpenTsDatasource extends DataSourceWithBackend<OpenTsdbQuer
       return of([]);
     }
 
+    if (config.featureToggles.opentsdbBackendMigration) {
```

Comment:
more of a nit but all of these are being made to the backend with double slashes:

<img width="336" height="74" alt="Screenshot 2025-12-15 at 6 56 20 PM" src="https://github.com/user-attachments/assets/afda48f9-6e0a-4c3d-a968-d1a5a01fcbbf" />

should we remove the leading `/` to prevent this?